components/dashboard/src/settings/Account.tsx

```diff
@@ -7,23 +7,40 @@
 import { User } from "@gitpod/gitpod-protocol";
 import { useContext, useState } from "react";
 import Modal from "../components/Modal";
+import { getGitpodService, gitpodHostUrl } from "../service/service";
 import { UserContext } from "../user-context";
 import { SettingsPage } from "./SettingsPage";
 
 export default function Account() {
     const { user } = useContext(UserContext);
 
     const [modal, setModal] = useState(false);
+    const [typedEmail, setTypedEmail] = useState('');
+
+    const primaryEmail = User.getPrimaryEmail(user!);
+
+    const deleteAccount = async () => {
+        await getGitpodService().server.deleteAccount();
+        //TODO we are sad to see you go view
+        document.location.href = gitpodHostUrl.asApiLogout().toString();
+    };
 
     const close = () => setModal(false);
     return <div>
         <Modal visible={modal} onClose={close}>
             <h3 className="pb-2">Delete Account</h3>
             <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-2">
-              <p className="mt-1 mb-2 text-base">Are you sure you want to delete your account? This action will remove all the data associated with your account in Gitpod and cannot be reversed.</p>
+                <p className="py-4 text-gray-900 text-base">You are about to permanently delete your account.</p>
+                <ol className="text-gray-400 text-sm list-outside list-decimal">
+                    <li className="ml-5">All your workspaces and related data will be deleted and cannot be restored afterwards.</li>
+                    <li className="ml-5">Your subscription will be cancelled. If you obtained a Gitpod subscription through the GitHub marketplace, you need to cancel your plan there.</li>
+                </ol>
+                <p className="pt-4 pb-2 text-gray-600 text-base font-bold">Type your email to confirm</p>
+              <input className="border-2 border-gray-400 w-full" type="text" placeholder={primaryEmail} onChange={e => setTypedEmail(e.target.value)}></input>
```

Comment:
**suggestion**: Maybe the specs were not clear enough. This shouldn't have any placeholder. 😇  /cc @AlexTugarev 

```suggestion
              <input className="border-2 border-gray-400 w-full" type="text" onChange={e => setTypedEmail(e.target.value)}></input>
```

Reply:
<img width="527" alt="Screen Shot 2021-03-23 at 14 32 48" src="https://user-images.githubusercontent.com/914497/112154670-ccd13880-8be4-11eb-94bb-60a9b1d80d13.png">

I'd like to copy the pattern. Others do that in a single line.

Reply:
@AlexTugarev I've cross-linked this conversation to the follow up issue in https://github.com/gitpod-io/gitpod/issues/3554#issue-838712657.